geomesa-gs-security/src/main/scala/org/geomesa/gs/security/audit/ElasticRequestDataListener.scala

```diff
@@ -0,0 +1,119 @@
+/***********************************************************************
+ * Copyright (c) 2013-2020 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the GNU GENERAL PUBLIC LICENSE,
+ * Version 2 which accompanies this distribution and is available at
+ * https://opensource.org/licenses/GPL-2.0.
+ ***********************************************************************/
+
+package org.geomesa.gs.security.audit
+
+import java.lang.reflect.Type
+import java.util.Date
+
+import com.google.gson._
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.http.HttpResponse
+import org.apache.http.client.config.RequestConfig
+import org.apache.http.client.methods.{HttpPost}
+import org.apache.http.concurrent.FutureCallback
+import org.apache.http.entity.StringEntity
+import org.apache.http.impl.nio.client.{CloseableHttpAsyncClient, HttpAsyncClients}
+import org.geoserver.monitor.{RequestData, RequestDataListener}
+import org.opengis.geometry.BoundingBox
+
+import scala.concurrent.Await
+
+
+class ElasticRequestDataListener extends RequestDataListener with LazyLogging {
+
+  private val requestConfig = RequestConfig.custom()
+    .setSocketTimeout(10000)
+    .setConnectTimeout(10000).build();
+
+  val client: CloseableHttpAsyncClient = HttpAsyncClients.custom()
+    .setMaxConnTotal(10)
+    .setDefaultRequestConfig(requestConfig)
+    .build()
+
+  client.start()
+  val host = sys.env.getOrElse("ELASTICSEARCH_HOST", null)
+  val index = sys.env.getOrElse("GEOSERVER_ES_INDEX", null)
```

Comment:
if these values are null, we should probably fail fast here, either disabling the listener or throwing an exception (which would kill the entire geoserver...)

Reply:
If it gets this far, it both will have a value, I just forgot to change it to a get instead of getOrElse